CHANGELOG.md

```diff
@@ -12,6 +12,22 @@
 
 ---
 
+[//]: # (START/v1.2.0)
+# v1.2.0
+
+## Features
+ * Return `label` property in ItemUrl object. {#66}
+ * Add functionality to fetch multiple vaults {#67}
+ * Add functionality to fetch single vault by title {#68}
+ * Three separate methods are available to fetch single vault: `getVault`, `getVaultById`, `getVaultByTitle` {#69}
+ * Add functionality to fetch multiple items by title {#70}
+ * Three separate methods are available to fetch a single item: `getItem`, `getItemById`, `getItemByTitle` {#71}
```

Comment:
This phrasing is inconsistent comparing to the others. I'd consider writing something like this:
```suggestion
 * Enable fetching a single item with three separate methods: `getItem`, `getItemById`, `getItemByTitle` {#71}
```

What could also help is say a short description for each method, since `getItem` for example is not that obvious only from the name. Something like this, perhaps?
```suggestion
 * Enable fetching a single item with three separate methods: 
   - `getItem` - get the item based on its ID or name
   - `getItemById` - get the item with the provided ID
   - `getItemByTitle` - get the item with the provided title. Note: The title has to be unique. If multiple items have the same title, consider getting the item by its ID instead. {#71}
```

What do you think?

Reply:
Sounds good! Changed